main/src/main/java/org/mobilitydata/gtfsvalidator/validator/StopLatLongValidator.java

```diff
@@ -0,0 +1,55 @@
+package org.mobilitydata.gtfsvalidator.validator;
+
+import org.mobilitydata.gtfsvalidator.annotation.GtfsValidator;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.notice.SeverityLevel;
+import org.mobilitydata.gtfsvalidator.notice.ValidationNotice;
+import org.mobilitydata.gtfsvalidator.table.GtfsLocationType;
+import org.mobilitydata.gtfsvalidator.table.GtfsStop;
+
+/**
+ * Validates {@code stops.stop_lat} and {@code stops.stop_long} exist for a single {@code GtfsStop}
+ * of type Stop, Station, Entrance, and Exit.
+ *
+ * <p>Generated notices:
+ *
+ * <ul>
+ *   <li>{@link LatLongRequiredForStopType}
+ * </ul>
+ */
+@GtfsValidator
+public class StopLatLongValidator extends SingleEntityValidator<GtfsStop> {
```

Comment:
I think it'd make sense to consolidate this logic into the existing StopLatLonValidator file.

Reply:
@bdferris-v2 this is probably due to me being new to the project... I was only aware of the validators in the [validator](https://github.com/MobilityData/gtfs-validator/tree/master/main/src/main/java/org/mobilitydata/gtfsvalidator/validator) folder... but doing a class search, I see a generated `GtfsStopLatLonValidator` class that I assume is generated from by `LatLonValidatorGenerator`.  As you mentioned on the call earlier, it looks like maybe it already covers this scenario, as it seems to [check](https://github.com/MobilityData/gtfs-validator/blob/master/processor/src/main/java/org/mobilitydata/gtfsvalidator/processor/LatLonValidatorGenerator.java#L96) if either coordinate is `<= 1.0` 

So now I'm wondering if I'm misunderstanding something, or if this change is unnecessary?

Reply:
Ah, duh, you are totally right, that's a generated validator :)  Sorry for the confusion, I was just doing a quick check.